client/src/modules/list/components/ItemsContainer.jsx

```diff
@@ -19,7 +20,8 @@ class ItemsContainer extends Component {
     const { items } = this.props;
 
     this.state = {
-      items
+      items,
+      displayedItemsCount: 0
```

Comment:
displayedItem~~s~~Count?

Reply:
I think it should stay plural cause it's about ITEMS not Item count. 
We count items, not a single item.

but I'm not a linguist :D

Reply:
https://english.stackexchange.com/a/11352 and https://english.stackexchange.com/questions/1314/user-accounts-or-users-account as a ref.

Reply:
yes, you're right :)